pkg/provisioner/ironic/ironic.go

```diff
@@ -1212,6 +1212,37 @@ func (p *ironicProvisioner) Provision(hostConf provisioner.HostConfigData) (resu
 		return transientError(provisioner.NeedsRegistration)
 	}
 
+	p.log.Info("WARNING: automated cleaning is disabled for host",
```

Comment:
Is this actually a warning?  Seems like it's just Info logging, since the behavior is what was requested?

In #787 I did also generate an event though, so it's visible via the BMH `describe` output, we may want the same in this case?

Reply:
Shouldn't this logging be inside some check?

Reply:
@hardys , @kashifest agree with both points. Updating. Thanks.